setup.py

```diff
@@ -29,19 +36,12 @@
         "Topic :: Security",
     ],
     install_requires=[
+        "pydpkg",
```

Comment:
Please move this change to https://github.com/canonical/sec-cvescan/pull/50/commits/1d15d9699a90a64ceb765062c83b69ec37467e6f

Reply:
🤦 Totally missed this bit, fixing